recipes/ckanweb-deploy-exts.rb

```diff
@@ -85,21 +85,28 @@
 #
 extordering =
 {
-	'odi_certificates' => 20,
-	'dcat structured_data' => 30,
-	'resource-visibility' => 35,
-	'data_qld_resources data_qld_integration data_qld_google_analytics data_qld_reporting' => 40,
-	'qgovext' => 45,
-	'datarequests' => 46,
-	'ytp_comments' => 47,
-	'scheming_datasets' => 50,
-	'qa' => 60,
-	'archiver' => 70,
-	'report' => 80,
-	'harvester_data_qld_geoscience' => 85,
-	'harvest' => 90,
-	'validation-schema-generator' => 95,
-
+    'xloader' => 10,
```

Comment:
@ThrawnCA  form you testing, ordering did not matter, should we pull this out or just run with it and see?

Reply:
Let's leave it in, it should be harmless. Yes, ordering didn't seem to affect testing in https://github.com/qld-gov-au/ckan-qld-infrastructure/actions/runs/4380618339